agentcore-strands-playground/br_utils.py

```diff
@@ -0,0 +1,61 @@
+"""
+Bedrock Utilities Module
+
+This module provides helper functions for interacting with Amazon Bedrock,
+including listing enabled foundation models in the configured AWS region.
+"""
+
+import os
+from typing import List, Dict, Optional
+import boto3
+from dotenv import load_dotenv
+
+# Load environment variables
+load_dotenv()
+
+def get_bedrock_models() -> List[str]:
+    """
+    Get a list of all enabled Bedrock foundation model identifiers.
+    
+    Returns:
+        List of model identifier strings.
+    """
+    models = [
```

Comment:
For future maintainability (not necessarily with this PR), suggest using Bedrock APIs to get the FM and filtering out only the ones you need based on the lifecycle of the model.. this will ensure that 6 months down the line, your code is still relevant